### PR TITLE
Improve error handling in $lib.stix.import.ingest (SYN-7837)

### PR DESCRIPTION
**Pull request by Cisphyx**

### CODE REVIEW

**Review by vEpiphyte — DISMISSED** [review]

File: changes/f2e24dc074f481337b9efedc07d20ddd.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+desc: Catch additional Python exceptions which could be raised by malformed input
+  to ``$lib.stix.import.ingest()`` and raise ``BadArg`` instead.
+prs:
+- 3867
+type: feat
```
Comment (vEpiphyte) [inline changes/f2e24dc074f481337b9efedc07d20ddd.yaml:6]: bug?

Comment (Cisphyx) [reply changes/f2e24dc074f481337b9efedc07d20ddd.yaml:6]: good point, will adjust

**Review by vEpiphyte — APPROVED** [review]

**Review by invisig0th — APPROVED** [review]
Approving, but is this something we should be using a JSON schema for?

**Comment by Cisphyx** [comment]
> Approving, but is this something we should be using a JSON schema for?

Good point, yeah a validator would probably be good here.

**Pull request merged**